nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Js tags user restrictions

# Description
Added nav bar, routes and add/remove tags to post restrictions on non-admin users.
Partially Fixes #34 
## Type of change
# Testing Instructions
- Checkout branch
- Make sure appsettings and sql express are updated
- Run API and App
- Login as an Author user
- Authors should NOT see or be able to access UserProfiles, Tag and Category Management via route manipulation
- In post details, Users should NOT be able to see the Manage Tags button on other user's post, additionally should NOT be able to manipulate the route in add/remove tags to post page to access another user's post tags
- All other functionalities outside of the above scope should work as normal
- Login as an Admin user
- Admins should have access to the UserProfiles, Tag and Category Management components
- Admins should NOT be able to modify another user's tags via post details or route manipulation
# Checklist:
